README.md

```diff
@@ -1413,4 +1413,9 @@ I'm [Yury](https://github.com/yuryzaytsev). This is a street cat named Bob. He i
 My name is [Dima](https://github.com/youhymuk) and I'm glad to study here with you.
 I wish you all success!
 
-![kottan](gif/cat-headshot.gif)
\ No newline at end of file
+![kottan](gif/cat-headshot.gif)
+
+## Hi! 
+My name is [Nastya](https://github.com/AsyaYeromina) Happy to be the part of such a cool comunity.
+**YOU ARE SO COOL, KOTTANS!!!**
+![cat-box](https://i.pinimg.com/564x/b5/1e/1b/b51e1b639edc459ac49b574ed69367ae.jpg)
```

Comment:
Perfect job!
It would be better if you setup your code editor to add empty line at the end of the file

Reply:
Oh, yes! Thanks for remark. 
Added an empty line in the next commit.

Reply:
Great job!